Makefile

```diff
@@ -29,16 +29,11 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
-GOVERSION=$(shell go version | cut -f3 -d' ' | cut -c3-)
+# Golang command for build
```

Comment:
Shall we add a comment specify that golang version has to be higher than 1.13?

Reply:
The whole project specifies go 1.16, so we are not officially supporting anything lower. With the upgrade to kubebuilder 3, it looks like the dependencies require at least go 1.15. The tools require only go 1.13, but I would rather leave it open and update to go 1.16.6 in https://github.com/grpc/grpc/pull/26881.